frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -484,9 +424,11 @@ const MultiStepWizard = (): JSX.Element => {
     if (parameter) {
       setTabIndex(1);
       if (!multiParams.length) {
-        setMultiParams([parameter as Data]);
+        setMultiParams([parameter as InternalState]);
```

Comment:
Is `multiParams`/`setMultiParams` just `internalState`, or is it also holding additional data?